lib/modules/idea-single-widgets/index.js

```diff
@@ -28,15 +28,40 @@ const fields = [
       choices: [
           {
               value: true,
-              label: "Yes"
+              label: "Yes",
+              showFields: ['shareChannelsSelection']
           },
           {
               value: false,
               label: "No"
           },
-      ],
-      def: true
-  }].concat(
+      ]
+  },
+    {
+        name: 'shareChannelsSelection',
+        type: 'checkboxes',
+        label: 'Select which share buttons you want to display',
+        choices: [
+            {
+                value: 'facebook',
+                label: "Facebook"
+            },
+            {
+                value: 'twitter',
+                label: "Twitter"
+            },
+            {
+                value: 'mail',
+                label: "E-mail"
+            },
+            {
+                value: 'whatsapp',
+                label: "Whatsapp"
+            },
+        ]
+    }

```

Comment:
Kunnen we hier ook een default op zetten? Want anders verdwijnen in bestaande sites de share buttons, klopt dat?